Sources/VLCCloudStorageController.h

```diff
@@ -8,6 +8,11 @@
 
 #import <Foundation/Foundation.h>
 
+typedef enum {
+    SORTINGMETHOD_NAME,
+    SORTINGMETHOD_MODIFIED_DATE
+} sortingMethod;
```

Comment:
What do you think about using `NS_ENUM` macro here? ([ref for modern Objective-C enum](https://developer.apple.com/library/archive/releasenotes/ObjectiveC/ModernizationObjC/AdoptingModernObjective-C/AdoptingModernObjective-C.html#//apple_ref/doc/uid/TP40014150-CH1-SW6))
Also, the name of the enum is a bit confusing, what would you think of something like `VLCCloudSortingCriteria...`.